Mathlib/Algebra/Module/Equiv.lean

```diff
@@ -322,10 +322,15 @@ variable {re₃₂ : RingHomInvPair σ₃₂ σ₂₃} [RingHomInvPair σ₃₁
 
 variable (e₁₂ : M₁ ≃ₛₗ[σ₁₂] M₂) (e₂₃ : M₂ ≃ₛₗ[σ₂₃] M₃)
 
--- Note: The linter thinks the `RingHomCompTriple` argument is doubled -- it is not.
+set_option linter.unusedVariables false in
 /-- Linear equivalences are transitive. -/
 @[trans, nolint unusedArguments]
```

Comment:
Please copy the comment from https://github.com/leanprover-community/mathlib/pull/18458 too

Reply:
This comment still applies

Reply:
I've mistaken this for the porting note, sorry.